megalinter/descriptors/php.megalinter-descriptor.yml

```diff
@@ -39,6 +40,8 @@ install:
           && chmod +x phive.phar \
           && mv phive.phar /usr/local/bin/phive \
           && rm phive.phar.asc
```

Comment:
Please can you put those instructions in the same RUN, to avoid another docker layer ? thanks :)